src/test/resources/org/broadinstitute/hellbender/utils/test/SamAssertionUtilsUnitTest/file1.sam

```diff
@@ -0,0 +1,25 @@
+@HD	VN:1.5	GO:none	SO:coordinate
```

Comment:
What is the new `expected.HiSeq.1mb.1RG.2k_lines.bqsr.DIQ.alternate.bam` for? It does not appear to be used, or to be related to this PR at all.